kratos/processes/calculate_discontinuous_distance_to_skin_process.h

```diff
@@ -461,11 +462,7 @@ class KRATOS_API(KRATOS_CORE) CalculateDiscontinuousDistanceToSkinProcess : publ
             << "Skin model part solution step data missing variable: " << rVariable << std::endl;
 
         // Initialize embedded variable value
-        #pragma omp parallel for
-        for (int i_elem = 0; i_elem < n_elems; ++i_elem) {
-            auto it_elem = mrVolumePart.ElementsBegin() + i_elem;
-            it_elem->SetValue(rEmbeddedVariable, rEmbeddedVariable.Zero());
-        }
+        VariableUtils().SetNonHistoricalVariable(rEmbeddedVariable, rEmbeddedVariable.Zero(), mrVolumePart.Elements());
```

Comment:
I think there is a similar fct to set the variable to Zero, please check

Reply:
Yes, Got one. SetNonHistoricalVariableToZero(). Replaced with it.